### ISSUE TITLE
Error: Could not find module `@ember/test-waiters` imported from `ember-google-maps/component-managers/map-component-manager`

### ISSUE THREAD

**Issue by glbatistabh** [sandydoo/ember-google-maps#130]
Hi. 
I created an empty ember project and followed the instructions on the site "https://ember-google-maps.sandydoo.me/docs/getting-started". But this one is wrong. Can you help me?

**step-by-step:**

1) `shell>ember new test-google-maps`
---
2) `shell>cd test-google-maps/`
---
3) `shell>ember install ember-google-maps`
---
4) `shell>edit config/environment.js`
![image](https://user-images.githubusercontent.com/8257020/127237528-1befa255-9b30-45a2-9f58-219315637a3b.png)
---
5) `shell>edit test-google-maps/app/templates/application.hbs`
![image](https://user-images.githubusercontent.com/8257020/127237606-9d017ff5-3af8-4add-8b9f-fadad36023fe.png)
---
6) `shell>ember server`
![image](https://user-images.githubusercontent.com/8257020/127237699-9b8a31e4-072b-4a52-9fcd-037affdfcabb.png)
---
7) show errors:
![image](https://user-images.githubusercontent.com/8257020/127237857-d806b32d-c71e-450c-a85d-854d86bcd779.png)
---
8) `shell>ember -v`
```
ember-cli: 3.17.0
node: 14.17.2
os: linux x64
```
---
9) `shell>cat package.json` 
```
{
  "name": "test-google-maps",
  "version": "0.0.0",
  "private": true,
  "description": "Small description for test-google-maps goes here",
  "repository": "",
  "license": "MIT",
  "author": "",
  "directories": {
    "doc": "doc",
    "test": "tests"
  },
  "scripts": {
    "build": "ember build --environment=production",
    "lint:hbs": "ember-template-lint .",
    "lint:js": "eslint .",
    "start": "ember serve",
    "test": "ember test"
  },
  "devDependencies": {
    "@ember/optional-features": "^1.3.0",
    "@glimmer/component": "^1.0.0",
    "@glimmer/tracking": "^1.0.0",
    "babel-eslint": "^10.0.3",
    "broccoli-asset-rev": "^3.0.0",
    "ember-auto-import": "^1.5.3",
    "ember-cli": "~3.17.0",
    "ember-cli-app-version": "^3.2.0",
    "ember-cli-babel": "^7.17.2",
    "ember-cli-dependency-checker": "^3.2.0",
    "ember-cli-eslint": "^5.1.0",
    "ember-cli-htmlbars": "^4.2.2",
    "ember-cli-inject-live-reload": "^2.0.2",
    "ember-cli-sri": "^2.1.1",
    "ember-cli-template-lint": "^1.0.0-beta.3",
    "ember-cli-uglify": "^3.0.0",
    "ember-data": "~3.17.0",
    "ember-export-application-global": "^2.0.1",
    "ember-fetch": "^7.0.0",
    "ember-google-maps": "^4.2.1",
    "ember-load-initializers": "^2.1.1",
    "ember-maybe-import-regenerator": "^0.1.6",
    "ember-qunit": "^4.6.0",
    "ember-resolver": "^7.0.0",
    "ember-source": "~3.17.0",
    "ember-welcome-page": "^4.0.0",
    "eslint-plugin-ember": "^7.7.2",
    "eslint-plugin-node": "^11.0.0",
    "loader.js": "^4.7.0",
    "qunit-dom": "^1.0.0"
  },
  "engines": {
    "node": "10.* || >= 12"
  },
  "ember": {
    "edition": "octane"
  }
}
```

**Comment by sandydoo** [comment]
Thanks, @glbatistabh. I've released [v4.2.2](https://github.com/sandydoo/ember-google-maps/releases/tag/v4.2.2) with a fix.
I'd recommend using a newer version of ember & ember-cli though!

**Comment by glbatistabh** [comment]
Thank you! Solved the problem...
